manager/cves_without_errata_handler.py

```diff
@@ -0,0 +1,24 @@
+"""
+Module for /cves_without_errata API endpoint
```

Comment:
Same as above, I'd rather call the file like `feature_handler.py` or something similar